src/Randomizer.Shared/Models/LogicConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel;
+
+namespace Randomizer.Shared
+{
+    public class LogicConfig
+    {
+        [DisplayName("Prevent Screw Attack Sock Locks"), Description("Requires the morph ball before using screw attacks in locations where you can get stuck"), Category("Logic")]
```

Comment:
Typo in "Sock Lock". I'd change the description to "You're not expected to use Screw Attack without the Morph ball" or something along those lines.

Reply:
What do you have against comfy socks? :(

Updated all of the display names/descriptions